docs/dev/Lua API.rst

```diff
@@ -1548,6 +1548,49 @@ Units module
 
   Returns a table of the cutoffs used by the above stress level functions.
 
+Action Timer API
```

Comment:
You should mention somewhere (probably where you talk about overflow protection) that if an action is in progress, the minimum value that it will be set to is 1 so that the action is completed on the next tick. Similarly, action types that are currently at 0 can't be set to any other value since there is no associated action.